src/main/java/com/kodilla/ecommercee/domain/Cart.java

```diff
@@ -1,21 +1,28 @@
 package com.kodilla.ecommercee.domain;
 
-import com.kodilla.ecommercee.dto.ProductDto;
 import lombok.AllArgsConstructor;
 import lombok.Getter;
 import lombok.NoArgsConstructor;
 
-import javax.persistence.Entity;
+import javax.persistence.*;
 import java.util.List;
 
 @AllArgsConstructor
 @NoArgsConstructor
 @Getter
 @Entity
+@Table(name = "CARTS")
 public class Cart {
 
+    @Id
+    @GeneratedValue
     private Long id;
-    private Long userId;
-    private List<ProductDto> productList;
+
+    @OneToOne
+    @JoinColumn(name = "userId")
+    private User user;
+
+    @OneToMany(targetEntity = Product.class, mappedBy = "cart", fetch = FetchType.LAZY)
```

Comment:
dodałbym cascade = CascadeType.All ,  aby mieć pewność , że obiekty które dodamy do listy productList pojawia sie w bazie danch. dopisanie tego parametru w nawiasach @OneToMany daje nam pewność co będzie sie działo z obiektem zależnym od tego obiektu w którym określamy te parametry.
Czyli np :
-jeżeli obiekt zostanie usunięty z bazy danych to usun wszystkie które są od niego zależne (aby nie było błędnych danych w bazie dancyh)
-jeżeli zostanie dodany obiekt do listy to automatycznie zostaje dodawany do bazy danych z odpowiednia zależnością.

Reply:
No właśnie, czy kasowanie koszyka powinna pociągnąć za sobą kasowanie produktów? w mojej opinii nie

Reply:
No racja , teraz jak to czytam drugi raz to nie ma sensu :)